ddl/db_integration_test.go

```diff
@@ -1933,6 +1933,16 @@ func (s *testIntegrationSuite5) TestChangingDBCharset(c *C) {
 	verifyDBCharsetAndCollate("alterdb2", "utf8mb4", "utf8mb4_general_ci")
 }
 
+func (s *testIntegrationSuite5) TestChangingUppercaseDBCharset(c *C) {
```

Comment:
Please merge this test case to existing cases:

https://github.com/pingcap/tidb/blob/7669c32bf8e3b184efe6cb42b98bd3d34a9602da/ddl/db_integration_test.go#L1831